### PR TITLE
Standardize goals and roadmaps for engineering teams

### PR DESCRIPTION
**Pull request by christinaforney**
Only tagged Eng managers for teams that had changes. No copy/goals/roadmap was changed other than moving the location.

This is expecting code intel PR to be merged: https://github.com/sourcegraph/about/pull/1952

### CODE REVIEW

**Review by chrispine — APPROVED** [review]
LGTM

File: handbook/engineering/campaigns/goals.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Campaigns goals and priorities
```
Comment (nicksnyder) [inline handbook/engineering/campaigns/goals.md:1]: Why is the title "goals and priorities" if the sections are "goals" and "roadmap"?

**Review by nicksnyder — APPROVED** [review]

File: handbook/engineering/campaigns/goals.md (hunk 1 above)
Comment (christinaforney) [reply handbook/engineering/campaigns/goals.md:1]: This is an artifact from before roadmap was added to this page. We can change it!

**Review by chayim — APPROVED** [review]
LGTM!

File: handbook/engineering/campaigns/goals.md (hunk 1 above)
Comment (christinaforney) [reply handbook/engineering/campaigns/goals.md:1]: I will follow up with PR to a change to all titles of this section after I get more feedback from folks.

**Pull request merged**